tests/harvest_api.py

```diff
@@ -190,11 +186,11 @@ def create_expense(project_id):
     #     receipt_file = {"harvest_test_receipt.gif": receipt}
```

Comment:
Any reason for keeping this commented code?

Reply:
It was existing commented code but removed it as it was unused.